content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+10 reasons you should give another chance to Firefox Nightly
+############################################################
+
+:slug: give-firefox-nightly-a-new-chance
+:date: 2016-12-22
+:lang: en
+:url: give-firefox-nightly-a-new-chance
+:summary:
+    Help us make the best Mozilla Firefox of all the times by mid 2017!
+:status: draft
+
+What is Nightly?
+================
+
+Firefox Nightly build also known as Nightly is a version of Firefox
+built from the source code everyday at night.
+
+This means that Firefox updates itself everyday!
+
+
+Why aren't you using Nightly today?
+===================================
+
+Anyone wants a stable Firefox and you don't have time for a crappy
+browser experience.
+
+Also, it was not always the case, Firefox Nightly is now much more
+stable than ever. The number of crashes has been schrinked down and it
+is getting lower really fast everyday.
+
+Also most of the bugs that you might have will be fixed the day after.
+
+You can always go in the `Help > About Firefox` menu to make sure you
+have the last update!
+
+
+Why you should actually give Firefox Nightly a try
+==================================================
+
+By installing Firefox Nightly you will:
+
+#. Have new Mozilla Firefox features 18 weeks before
+#. Have a much faster Firefox
+#. Make a difference
+#. Help reduce the number of crashes for Mozilla Firefox users
+#. Make sure Mozilla Firefox source code is in a workable state
+#. Help add-ons developer to validate that their add-ons is still working
+#. Have a better developer experience to develop your apps
+#. Give a really valuable contribution to the Mozilla project without coding
+#. Participate at building an Free Libre and Open Source browser that respect user privacy
+#. Have a real impact on how the Web is built noawadays.
+
+Have a much faster Firefox
+--------------------------
+
+Firefox is getting smaller new features are now tested through add-ons
+using `Test Pilot <https://testpilot.firefox.com/>`_ instead of
+getting added to Firefox.
```

Comment:
I don't really understand what you are trying to say by `getting added to Firefox` here. Instead of being shipped with Firefox? And then I don't really understand the flow of logic from here to talking about Quantum, which is a much bigger thing than can be shipped with Test Pilot.